frontend/src/pages/modelServing/screens/global/InferenceServiceStatus.tsx

```diff
@@ -101,20 +101,21 @@ const InferenceServiceStatus: React.FC<InferenceServiceStatusProps> = ({
     }
   };
 
+  const bodyContent = modelStatus?.failedToSchedule ? (
+    <Text>Insufficient resources</Text>
+  ) : (
+    <Text>{getInferenceServiceStatusMessage(inferenceService)}</Text>
+  );
```

Comment:
You cannot include `Text` in both paths if you want to use this later for a truthy check because otherwise it will always be truthy. Capture just the data here instead:
```suggestion
  const bodyContent = modelStatus?.failedToSchedule ? (
    'Insufficient resources'
  ) : (
    getInferenceServiceStatusMessage(inferenceService)
  );
```